### PR TITLE
Show raster row/column of clicked point in identify results

### LINKED ISSUE #8500: Show raster row/column of clicked point in identify results (Request in QGIS)
### Request for documentation
From pull request QGIS/qgis#54525
Author: @nyalldawson
QGIS version: 3.34

**Show raster row/column of clicked point in identify results**

### PR Description:
Resurrects https://github.com/qgis/QGIS/pull/52804, gets tests passing

### Commits tagged with [need-docs] or [FEATURE]

### PR DESCRIPTION
**Pull request by selmaVH1**
Fixes #8500 
I believe that such 'small details' should be mentioned in the 'Working with Raster' chapter, but I'm not sure how to incorporate them into that chapter. I would like to hear suggestions and opinions on this.

### CODE REVIEW

**Comment by DelazJ** [comment]
Interesting remark. The identify tool description is quite vector centered right now despite it can be used also with raster, mesh or point cloud. We could move/create a section for results dialog (and format specifics) to each data type chapter, leaving the common how-to parts and options description in the general tools chapter.
About where to place such info in the raster chapter: probably along with the properties because I expect the analysis page to go when the raster catalog tool gets removed (according to [QEP 271](https://github.com/qgis/QGIS-Enhancement-Proposals/issues/271)), and georeferencer should [move out from raster chapter](https://github.com/qgis/QGIS-Documentation/issues/7384#issuecomment-1739451211). We could have an intro section as done for mesh/point cloud/tiles formats (why not?) and interacting with raster using the identifier could be an element... 
This location could be applied to mesh and point cloud also, and for vector, maybe should it go to attribute table/forms page (?).

Just a rough idea (unfortunately not resulting in a single line PR 😄) and I'm open to all options...

**Comment by selmaVH1** [comment]
> About where to place such info in the raster chapter: probably along with the properties because I expect the analysis page to go when the raster catalog tool gets removed (according to [QEP 271](https://github.com/qgis/QGIS-Enhancement-Proposals/issues/271)), and georeferencer should [move out from raster chapter](https://github.com/qgis/QGIS-Documentation/issues/7384#issuecomment-1739451211). We could have an intro section as done for mesh/point cloud/tiles formats (why not?) and interacting with raster using the identifier could be an element...

I have added the "Introduction to Raster Data" chapter and provided details about the Identify tool. I will create new PRs for point cloud, mesh, and vector data.

File: docs/user_manual/working_with_raster/raster_properties.rst [hunk 1]
```diff
@@ -1,9 +1,43 @@
 .. index:: Raster, Layer properties
+
+**************************
+Intoduction to Raster Data
```
[the remark this answers is not in the document]
Comment (selmaVH1) [reply docs/user_manual/working_with_raster/raster_properties.rst:4]: I agree...  Is it possible to retain the text but remove the sectioning (title), if that makes sense? I was considering removing it entirely...

[the remark this answers is not in the document]
Comment (DelazJ) [reply docs/user_manual/working_with_raster/raster_properties.rst:4]: Assuming I understood correctly, I'd find weirder to have a new chapter/page that starts with no title.

**Pull request merged**